tmpl/wiki/opam.md

```diff
@@ -1,6 +1,6 @@
 ### Maintenance
 
-The `opam upgrade` command will refresh all your remote repositories, and recompile any outdated libraries. You will need to run this once per compiler installed, so switch between them.
+The `opam update` command will refresh all your remote repositories. The `opam upgrade` command will recompile any outdated libraries. You will need to run this once per compiler installed, so switch between them.
```

Comment:
Also the instructions use `opam remote`, this should be updated to use `opam repository`.